src/hooks/use-schema-loader.js

```diff
@@ -1,15 +1,20 @@
 import { useState } from 'react';
 
 const useSchemaLoader = initialPath => {
-  const [paths, setPaths] = useState([{ forPath: initialPath }]);
+  const initialState = [{ forPath: initialPath }];
```

Comment:
Serious question: why put `forPath` in an object?

Reply:
the default `forPath` is an empty array, if you spread that out it gets lost. Seemed like an easy solution to just make an object with a property, on the schema-menu I'm also adding a "title" to that object as well.